package.json

```diff
@@ -18,7 +18,7 @@
     "lodash": "^3.6.0",
     "mime": "^1.3.4",
     "napa": "^1.2.0",
-    "protagonist": "^0.19.0",
+    "protagonist": "^1.3.3",
```

Comment:
Actually it should be in a different PR. (if not this update will break the aglio part). In the PR we should also get rid of the serve command and corresponding code